package.json

```diff
@@ -52,7 +52,7 @@
     "safe-buffer": "5.1.2",
     "send": "0.17.1",
     "serve-static": "1.14.1",
-    "setprototypeof": "1.1.1",
+    "setprototypeof": "1.2.0",
```

Comment:
I think this is already in #4011

Reply:
Yes! Seems duplicated. Do you want me to revert it?

Reply:
We may want to so as I begin landing these once the plan is determined it doesn't delay with merge conflicts.